index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <div class="toolbar">
+    <header>
+      <a class="my-logo" href="#">
+        <img src="/images/My-Logo.png" alt="My Logo icon.">
+      </a>
+    </header>
+      <nav>
+        <a class="hamburger" href="#">
+          <img src="/images/Hamburger.png" alt="Hamburger icon.">
+        </a>
+      </nav>
+    </div>
+      <div class="background">
+    <main class="headline">
+      <h2 class="primarytext">I’m Marcos
```

Comment:
- [ ] Following semantic HTML, there'd be no need for the `<div class="background">`. Please remove it. 
- [ ] Also, the`<main class="headline">` tag should be a `section` tag instead because it represents the headline section of your web page.
- [ ] The main title "I'm Marcos Glad to see you" should be a `<h1>` as it is the main heading of your page

Reply:
@codepantha Thank you for replying to me! Does the headline represent all of the content below the header? (I mean with the social media ul and everything)

Reply:
yes, it does